src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -511,7 +511,7 @@ void program::query_local_block_io_supported() {
         auto _kernels_cache_device_query = std::unique_ptr<kernels_cache>(new kernels_cache(_engine, prog_id,
                                                                     kernel_selector::KernelBase::get_db().get_batch_header_str()));
         auto id = _kernels_cache_device_query->set_kernel_source(kernel_string, false);
-        _kernels_cache_device_query->build_all();
+        _kernels_cache_device_query->build_all(true);
```

Comment:
could you add a comment why it is `true` here?

Reply:
This parameter is removed. Thank you.